modules/services/repositories/src/main/java/au/com/shiftyjelly/pocketcasts/repositories/podcast/PlaylistManagerImpl.kt

```diff
@@ -309,6 +319,35 @@ class PlaylistManagerImpl @Inject constructor(
         return playlistDao.findNotSynced()
     }
 
+    private fun sendPlaylistUpdateAnalytics(
+        playlist: Playlist,
+        userPlaylistUpdate: UserPlaylistUpdate?
+    ) {
+        // Don't send a filter updated event if the playlist is being created or if
+        // the user did not initiate the update
+        if (!playlist.draft && userPlaylistUpdate != null) {
+            userPlaylistUpdate.properties.map { playlistProperty ->
+                when (playlistProperty) {
+
+                    is FilterUpdatedEvent -> {
+                        val properties = mapOf(
+                            "group" to playlistProperty.groupValue,
+                            "source" to userPlaylistUpdate.source.analyticsValue
```

Comment:
"group" and "source" are extracted to constants in the follow-up PR: https://github.com/Automattic/pocket-casts-android/pull/386